### PR TITLE
Mod to setup.sh to allow inserting self signed or other non root-CA certs (tested on ubuntu only)

### PR DESCRIPTION
**Pull request by hugh-mend**

### CODE REVIEW

File: Repo-Integration/setup.sh [hunk 1]
```diff
@@ -85,7 +152,8 @@ then
     echo "${cyn}export ws_key='replace-with-your-activation-key-inside-single-quotes'${end}"
     exit
 else
-    scm
+    scm;
+    cert_add;
```
Comment (samq-ws) [inline Repo-Integration/setup.sh:156]: Cert_add should only fire if $2 is supplied

**Review by samq-ws — CHANGES_REQUESTED** [review]
Please add information to readme on how to use this new parameter.

File: Repo-Integration/setup.sh (hunk 1 above)
Comment (hugh-mend) [reply Repo-Integration/setup.sh:156]: This is taken care of by first if/fi statement in the cert_add function

**Comment by hugh-mend** [comment]
@samq-ws I updated the readme file.  Sorry for not squashing the commits ahead of time; will do next time.

File: Repo-Integration/setup.sh [hunk 2]
```diff
@@ -77,6 +81,69 @@ echo "${grn}Download Success!!!${end}"
 
 }
 
+function cert_add(){
+    #SUPPORTEDVERSION=22.12.2
+    if [ -z $CERTFILE ]
+        then  echo "No .crt file supplied as 2nd argument. Integration will be prepared with no additional certs."
+        else
+            # if [[ $AGENT_LATEST != *$SUPPORTEDVERSION/ ]]
```
Comment (samq-ws) [inline Repo-Integration/setup.sh:89]: remove comments

**Review by samq-ws — CHANGES_REQUESTED** [review]
remove comments as pointed out in review

**Review by samq-ws — APPROVED** [review]

**Pull request merged**